### PR TITLE
APR-1281 and APR-1280- section 7.4 – Update

### PR DESCRIPTION
**Pull request by MakCom**

### CODE REVIEW

File: src/SFA.DAS.QnA.Database/projects/roatp/project.json [hunk 1]
```diff
@@ -220,6 +220,12 @@
                "SequenceNo":7,
                "SectionNo":4,
                "name":"Quality and high standards in apprenticeship training"
+            },
+			{
```
Comment (David-B-Read) [inline src/SFA.DAS.QnA.Database/projects/roatp/project.json]: Can you fix the indenting, and the trailing space in the section name?

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.4.json [hunk 2]
```diff
@@ -51,14 +55,74 @@
...
+            "LinkTitle": "Tell us how these expectations for quality and high standards in apprenticeship training are monitored and evaluated",
+            "InfoText": "",
+            "Questions": [
+                {
+                    "QuestionId": "DAT-7305",
+                    "QuestionTag": null,
+                    "Label": "Tell us how these expectations for quality and high standards in apprenticeship training are monitored and evaluated",
+                    "ShortLabel": "",
+                    "InputClasses": null,
+                    "QuestionBodyText": "<p class='govuk-body'>Your answer must include how they're:</p><ul class='govuk-list govuk-list--bullet'><li>reviewed and monitored</li><li>reflected in developing and delivering apprenticeship training programmes</li><li>measured through key performance indicators for the outcomes of apprenticeship training programmes</li><li>used to benchmark apprenticeship training delivery against industry leaders</li</ul>",
```
Comment (David-B-Read) [inline src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.4.json]: Last close LI tag is missing a '>'

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.4.json [hunk 3]
```diff
@@ -51,14 +55,74 @@
...
+            "Feedback": null,
+            "ActivatedByPageId": "7300"
+        },
         {
             "PageId": "7310",
             "SequenceId": null,
             "SectionId": null,
             "Title": "Tell us who's responsible for maintaining these expectations for quality and high standards in apprenticeship training",
-            "LinkTitle": "",
+            "LinkTitle": "Responsible for high standards",
```
Comment (David-B-Read) [inline src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.4.json]: Link title should match the question/page title, this is used in the assessor views

File: src/SFA.DAS.QnA.Database/projects/roatp/project.json (hunk 1 above)
Comment (MakCom) [reply src/SFA.DAS.QnA.Database/projects/roatp/project.json]: I was using Notepad++, which seems that renders the indenting differently than VS. It’s fixed now.

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.4.json (hunk 2 above)
Comment (MakCom) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.4.json]: My bad. Fixed it.

File: src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.4.json (hunk 3 above)
Comment (MakCom) [reply src/SFA.DAS.QnA.Database/projects/roatp/sections/section7.4.json]: It’s from the previous version, but fixed it.

**Pull request merged**